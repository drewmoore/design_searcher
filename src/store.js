import { createStore, applyMiddleware } from 'redux';
import thunk from 'redux-thunk';
import root  from './reducers/root';

const middleWare = applyMiddleware(thunk);
const store      = createStore(root, {}, middleWare);

export default store;
