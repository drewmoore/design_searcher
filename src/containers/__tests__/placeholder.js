import React                from 'react';
import { Provider }         from 'react-redux';
import renderer             from 'react-test-renderer';
import { createStore }      from 'redux';
import { mount }            from 'enzyme';
import store                from '../../store';
import PlaceholderContainer from '../placeholder';

describe('PlaceholderContainer', () => {
  let template;

  beforeEach(() => {
    template = (
      <Provider store={store}>
        <PlaceholderContainer />
      </Provider>
    );
  });

  describe('DOM Snapshot', () => {
    test('matches snapshot', () => {
      const tree = renderer.create(template).toJSON();
      expect(tree).toMatchSnapshot();
    })
  });

  describe('Rendering Props from State', () => {
    test('renders message', () => (
      expect(mount(template).text()).toMatch(/World/)
    ));
  });
});
