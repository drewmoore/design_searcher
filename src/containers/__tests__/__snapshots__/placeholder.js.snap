// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaceholderContainer DOM Snapshot matches snapshot 1`] = `
<h1>
  Yo, World!
</h1>
`;
